libraries/core-react/rollup.config.js

```diff
@@ -49,10 +55,7 @@ export default [
         sourcemap: true,
         globals,
       },
-      {
-        file: pkg.main,
-        format: 'cjs',
-      },
+      ...(buildForStorybook ? [] : [{ file: pkg.main, format: 'cjs' }]),
```

Comment:
Why do we not build the `commonjs` dist for storybook?

Reply:
Because the storybook don’t use that file so it’s a waste of time